libnmstate/nm/ipv6.py

```diff
@@ -158,3 +163,24 @@ def get_ip_profile(active_connection):
     if remote_conn:
         return remote_conn.get_setting_ip6_config()
     return None
+
+
+def get_route_running():
+    client = nmclient.client()
+    skip_networks = [iplib.IPV6_LINK_LOCAL_NETWORK,
+                     iplib.IPV6_KERNEL_AUTO_MULTICAST_NETWORK]
+    routes = nm_route.get_running(
+        client,
+        nmclient.NM.ActiveConnection.get_ip6_config,
+        skip_networks)
+    for route in routes:
+        if route[LinuxRoute.METRIC] == 0:
+            # Kernel will convert 0 to IPV6_DEFAULT_ROUTE_METRIC.
```

Comment:
Isn't this a NM bug then?

Reply:
That's from NM document.

Reply:
Note that for IPv6, the kernel accepts zero (0) but coerces it to 1024 (user default). Hence, setting this property to zero effectively mean setting it to 1024. For IPv4, zero is a regular value for the metric.

Reply:
Sorry, I do not understand why this is not an NM bug.
I would expect NM to return 1024 and not 0, because 0 is not a valid number when reading it from the kernel.
If someone sets 0 in the NM config, it should wither fail or save it as 1024 implicitly (like the kernel does).

Reply:
I fixed our code to only check config routes for zero IPv6 metric and tested.

Bug reported at: https://bugzilla.redhat.com/show_bug.cgi?id=1692911